chunky/src/java/se/llbit/chunky/renderer/scene/Scene.java

```diff
@@ -310,6 +311,7 @@ public Scene() {
     width = PersistentSettings.get3DCanvasWidth();
     height = PersistentSettings.get3DCanvasHeight();
     sppTarget = PersistentSettings.getSppTargetDefault();
+    sceneDirectory = new File(PersistentSettings.getSceneDirectory() + File.separator + name);
```

Comment:
Yup, maybe move this to the `RenderContext`. The sceneDirectory doesn't belong in the scene.